src/java/org/apache/ivy/plugins/resolver/IBiblioResolver.java

```diff
@@ -191,7 +191,34 @@ public void endElement(String uri, String localName, String qName)
                         
                         return rev;
                     }
-                } else {
+                } else if (root.startsWith("file:")) {
```

Comment:
I'm inferring from this that `maven-metadata-local.xml` is only a thing in local repos. Sorry, not a part of maven repos that I knew about.